lldb/packages/Python/lldbsuite/test/tools/lldb-dap/dap_server.py

```diff
@@ -591,6 +591,7 @@ def request_attach(
         attachCommands=None,
         terminateCommands=None,
         coreFile=None,
+        stopOnAttach=True,
```

Comment:
Is there any reason we want to make this true by default?
1) Our current tests don't have stopOnEntry by default
2) DAP doesn't have definition about this nor does lldb-dap documentation. So most real life attach case will not have this turned on.
3) I don't think we have lldb-dap test case covering multithreading scenario. But on linux, ptrace will put SIGSTOP on every thread and currently in `SendThreadStoppedEvent(dap)` we will send a `stopped` event for any threads that has the a stop reason, meaning we will actually send N stopped event (N being # of threads) upon attach request with `stopOnAttach` set to true.

Though it's only turned on for tests, I think we should make it false until it become the expected default behavior.

Reply:
See my reply to John: https://github.com/llvm/llvm-project/pull/138219#discussion_r2072197289